src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
+   * @typedef {object} PersonObject
+   * @param {string} id The person identifier
+   * @param {Array.<String>} emails An array of emails for the person
+   * @param {string} displayName The name to be displayed for the person
+   * @param {string} firstName The first name of the person
+   * @param {string} lastName The last name of the person
+   * @param {string} nickName The short name for the person
+   * @param {string} avatar The full url to the person's avatar
+   * @param {string} orgId The id of the organization the person belongs to
+   * @param {string} status The presence status of the user.
+   *                        One of: '', 'active', 'bot', 'call', 'dnd', 'group', 'inactive', 'meeting', 'ooo', 'presenting', 'self', 'typing'
```

Comment:
We should probably document the only the ones we will support, and update if/when we support more states. From Jira specs there are:
* `null` -> no state, (open for discussion on this one because the API returns `'unknown'` for no state)
* `self`
* `active`
* `bot`
* `dnd`
* `ooo`

Reply:
Will do. I just grabbed these from momentum

Reply:
Actually, was there a reason for not supporting all the types that momentum supports?

Reply:
I guess I was thinking about it in terms of what we will implement and how we get the data. There are some status that we won't be supporting with the MVP of the SDK adapter, like `typing`, but it really doesn't affect the avatar's ability to do it since Momentum supports it. You can ignore my comment